### PR TITLE
[zh-cn]: update for color-interpolation-method

### PR DESCRIPTION
**Pull request by fuchunhui**

### CODE REVIEW

File: files/zh-cn/web/css/color-interpolation-method/index.md [hunk 1]
```diff
@@ -1,31 +1,38 @@
...
+  - : 关键字 `srgb`、`srgb-linear`、`display-p3`、`a98-rgb`、`prophoto-rgb`、`rec2020`、`lab`、`oklab`、`xyz`、`xyz-d50` 或 `xyz-d65` 之一。
+
 - `<polar-color-space>`
-  - : 关键字 `hsl`、`hwb`、`lch` 或 `oklch`。
+
+  - : 关键字 `hsl`、`hwb`、`lch` 或 `oklch` 之一。
+
 - {{CSSXref("&lt;hue-interpolation-method&gt;")}} {{optional_inline}}
+
   - : 用于色相插值的算法。默认为 `shorter hue`。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/css/color-interpolation-method/index.md:37]:
```suggestion

- `<custom-color-space>`
  - : 一个指向自定义 [@color profile](/zh-CN/docs/Web/CSS/@color-profile) 的 [`<dashed-ident>`](/zh-CN/docs/Web/CSS/dashed-ident#using_with_color-profile)。

```

File: files/zh-cn/web/css/color-interpolation-method/index.md [hunk 2]
```diff
@@ -1,31 +1,38 @@
...
```
Comment (JasonLamv-t) [inline files/zh-cn/web/css/color-interpolation-method/index.md:36]: Better to use same word for same concept. Keep uniformity.
```suggestion
  - : 用于色调插值的算法。默认为 `shorter hue`。
```

**Review by JasonLamv-t — DISMISSED** [review]
Well done! Just two nits.

File: files/zh-cn/web/css/color-interpolation-method/index.md [hunk 3]
```diff
@@ -112,7 +79,70 @@ in <polar-color-space>[ <hue-interpolation method>]
...
+.radial {
+  background-image: repeating-radial-gradient(
+    in oklch longer hue,
+    blueviolet 50px,
+    burlywood 100px
+  );
+}
+```
+
+#### Result
```
Comment (yin1999) [inline files/zh-cn/web/css/color-interpolation-method/index.md:141]:
```suggestion
#### 结果
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**